frontend/app/login/page.jsx

```diff
@@ -11,21 +11,70 @@ const Login = () => {
 
     const signIn = async () => {
         try {
-            const { user: { uid } } = await signInWithPopup(auth, provider);
+            const {
+                user: { uid },
+            } = await signInWithPopup(auth, provider);
             Cookies.set('uid', uid);
-            router.push('/dashboard'); 
-        } catch(err) {
+            router.push('/teams');
+        } catch (err) {
             console.log(err);
         }
-    }
-    
+    };
+
     return (
         <div className={loginStyles.container}>
-            <button onClick={() => signIn()}>
-                 { `G -> Sign in with Google` }
-            </button>
+            <div className={loginStyles.blurbBackground}>
+                <div className={loginStyles.blurbContainer}>
+                    <div className={loginStyles.headerContainer}>
+                        <h1 className={loginStyles.headerAssistant}>Assistant</h1>
+                        <div className={loginStyles.underline}></div>
+                        <h1>Captain</h1>
+                    </div>
+                    <p>
+                        The app that let's you create teams and manage rosters. Send SMS to confirm
+                        upcoming game rosters. View team schedules and build rosters according to
+                        your needs.
+                    </p>
+                </div>
+            </div>
+            <div className={loginStyles.loginContainer}>
+                <div className={loginStyles.login}>
+                    <h4>{`Login :)`}</h4>
```

Comment:
You skipped from an H1 to an H4 here. Normally I would suggest using an H2 or H3 here, but in this case this login text could likely just be a 'p' element. 

https://semji.com/blog/why-use-h1-h2-h3-tags-for-your-seo/#:~:text=The%20structure%20of%20H1%2C%20H2%2C%20H3%20tags,-Keep%20in%20mind&text=For%20an%20article%20or%20web,information%2C%20with%20more%20details.